src/Services/AllergyIntoleranceService.php

```diff
@@ -83,15 +82,17 @@ public function getAll($search = array(), $isAndCondition = true)
 
         $sqlBindArray = array();
         $sql = "SELECT lists.*,
-        us.uuid as practitioner,
+        users.uuid as practitioner,
+        facility.uuid as organization,
```

Comment:
since now including facility in this, need to ensure the facility uuid's are all populated in the __construct function in this class (see how is done for other tables there)

Reply:
I will add the same. Will keep this in notice for future implementations.

Reply:
Implemented the same now.